### PR TITLE
add config options for speeding up tests

### PR DESCRIPTION
**Pull request by mixmix**

### CODE REVIEW

File: test/basic.js [hunk 1]
```diff
@@ -29,14 +29,19 @@ rimraf.sync(dir)
...
 
 const keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+const config = {
+  keys,
+  path: dir,
+  db2: {
+    flushDebounce: 10,
+    writeTimeout: 10
+  }
+}
```
Comment (mixmix) [inline test/basic.js:39]: this config change took the run time of this file from 14s down to 4s .....

Comment (mixmix) [reply test/basic.js:39]: <future> someone should make a test/helpers/testbot.js so it's easy to apply this win across the whole repo....

File: indexes/plugin.js [hunk 2]
```diff
@@ -108,7 +108,8 @@ module.exports = class Plugin {
      */
     this.forcedFlush = this._flush.bind(this, true)
 
-    const liveFlush = debounce(this.flush, 250)
+    const flushDebounce = (configDb2 && configDb2.flushDebounce) || 250
+    const liveFlush = debounce(this.flush, flushDebounce)
```
Comment (staltz) [inline indexes/plugin.js:112]: Totally optional suggestion:

```suggestion
    const liveFlush = debounce(this.flush, configDb2?.flushDebounce ?? 250)
```

**Review by staltz — APPROVED** [review]
Yup, this is great, thanks Mix.

File: indexes/plugin.js (hunk 2 above)
Comment (mixmix) [reply indexes/plugin.js:112]: skipping because node 12 will cry :D

**Pull request merged**